config/locales/dotiw/en.yml

```diff
@@ -2,4 +2,4 @@
 en:
   datetime:
     dotiw:
-      two_words_connector: and
+      two_words_connector: " and"
```

Comment:
this is the bit about spacing I think could get messed up easily --- I'd see if we can remove the leading space here and just do it in the ruby part like